Builds/Ubuntu/install_boost.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+#
+# This script builds boost with the correct ABI flags for ubuntu
+#
+
+if hash lsb_release 2>/dev/null; then
+    ubuntu_release=$(lsb_release -sr)
+fi
+
+if [ -z "${ubuntu_release}" ]; then
+    echo "System not supported"
+    exit 1
+fi
+
+extra_defines=""
+if [ ${ubuntu_release} != "15.10" ]; then
+    extra_defines="define=_GLIBCXX_USE_CXX11_ABI=0"
+fi
+
+curl -L "http://downloads.sourceforge.net/project/boost/boost/1.59.0/boost_1_59_0.tar.bz2?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fboost%2Ffiles%2Fboost%2F1.59.0%2F&ts=1445463403&use_mirror=iweb" -o boost_1_59_0.tar.bz2
```

Comment:
@MarkusTeufelberger Those files don't work for me. `bootstrap.sh` won't run, and the tarballs don't seem to include the sources (they are only ~ 1M and the usual tarball is ~80M). Do you how I install from those tarballs? (I'm guessing they are not meant to be installed from, but I'd love to be wrong).